### ISSUE TITLE
Timeout error messages aren't really clear

### ISSUE THREAD

**Issue by rock3r** [actionsflow/actionsflow#7]
## Description

When a trigger times out, the error message looks like this:

```js
Actionsflow CLI version: 1.1.3

[14:54:45] INFO actionsflow: Clean the dest folder
[14:54:45] INFO actionsflow: No new updates found at trigger [rss] of workflow [01.yml], skip it.
[14:54:46] INFO actionsflow: No new updates found at trigger [rss] of workflow [02.yml], skip it.
[14:55:46] ERROR Actionsflow-trigger [rss]: fetch rss feed [https://someblog.com/feed/] error:  Error: Request timed out after 60000ms
    at Timeout.<anonymous> (/home/runner/work/_actions/actionsflow/actionsflow-action/v1/dist/main/index.js:1:661691)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
[14:55:47] INFO actionsflow: No new updates found at trigger [rss] of workflow [04.yml], skip it.
[14:55:48] INFO actionsflow: No new updates found at trigger [rss] of workflow [05.yml], skip it.
[14:55:48] INFO actionsflow: No new updates found at trigger [rss] of workflow [06.yml], skip it.
[14:55:48] INFO actionsflow: No new updates found at trigger [rss] of workflow [07.yml], skip it.
[14:55:48] INFO actionsflow: No new updates found at trigger [rss] of workflow [08.yml], skip it.
[14:55:49] INFO actionsflow: No new updates found at trigger [rss] of workflow [09.yml], skip it.
[14:55:49] INFO actionsflow: When running trigger [rss], an error occured Error: Request timed out after 60000ms
    at Timeout.<anonymous> (/home/runner/work/_actions/actionsflow/actionsflow-action/v1/dist/main/index.js:1:661691)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
[14:55:49] INFO actionsflow: Done.
``` 
But it's not clear from the log that the last error entry represents the same issue as the first error entry, nor either error entry mentions the flow file it was running when it caused the issue. Not sure if it's an issue limited to the rss trigger.
 
### Steps to reproduce

In my case, the issue is an RSS feed trigger that times out for whatever reason (the feed is there, when checking from my computer).

### Expected result

The first error message should say the name of the flow file it was running, and the last error message, which I assume is a recap of the errors that happened, should make it clear it's about recapping the errors that occurred during the run.

### Actual result

See log above.

**Comment by theowenyoung** [comment]
@rock3r Thanks, I'll print the workflow information when the error occur at the last error summary.

The first error output printed by the trigger [rss].

**Comment by rock3r** [comment]
Ace, thanks @theowenyoung - loving actionsflow so far!

**Comment by theowenyoung** [comment]
Thanks @rock3r , I have just released 1.1.4. (you don't need to upgrade manually, `v1` is always point to the latest release version)

You can rerun your workflow for get a more detailed error log like this:

```bash
16:44:27] INFO actionsflow: Actionsflow build finished with the following errors:
[16:44:27] INFO actionsflow: When running trigger [telegram_bot] of workflow file [telegram.yml]:  Error: Miss param token!
    at TelegramBot.run (/home/runner/work/workflow-for-test/workflow-for-test/node_modules/@actionsflow/trigger-telegram_bot/dist/src/index.js:34:19)
    at Object.exports.run (/home/runner/work/workflow-for-test/workflow-for-test/node_modules/actionsflow/dist/src/trigger.js:100:81)
    at Object.build [as handler] (/home/runner/work/workflow-for-test/workflow-for-test/node_modules/actionsflow/dist/src/build.js:104:48)
```

Let me know if you have any further question/issue. Thanks!

**Comment by rock3r** [comment]
That's great, thanks again for the super quick resolution, too. I really appreciate it.
